src/main/java/lotto/controller/GameController.java

```diff
@@ -0,0 +1,30 @@
+package lotto.controller;
+
+import lotto.domain.Computer;
+import lotto.domain.GameResult;
+import lotto.Lotto;
+import lotto.domain.Player;
+
+import java.util.List;
+
+public class GameController {
+    private final Player player;
+    private final Computer computer;
+    private final GameResult gameResult;
+
+    public GameController() {
+        player = new Player();
+        computer = new Computer();
+        gameResult = new GameResult();
```

Comment:
```suggestion
        gameResult = new GameResult(lottos, winningNumbers, bonusNumber);
```

객체의 일관성을 위해 
객체를 만든뒤 calculateTotalResult를 호출하여 값을 구하는것보다 
생성자를 이용해 값을 구하도록 설계를 수정해보는건 어떨까요?

참고: https://velog.io/@hope1213/Setter-%EC%82%AC%EC%9A%A9%EC%9D%84-%EC%99%9C-%EC%A7%80%EC%96%91%ED%95%B4%EC%95%BC%ED%95%A0%EA%B9%8C

Reply:
@kjb95 기존 방식이 왜 객체의 일관성을 저해하는지 잘 이해가 가지 않습니다😢 객체가 생성될 때 해당 객체가 해야 하는 일들을 전부 다 할 수밖에 없도록 하는 것이 포인트인가요?

Reply:
calculateTotalResult 메소드는 public으로 선언되었기 때문에 어디에서나 호출될수 있습니다.
따라서 GameResult의 result 상태는 언제든지 변경될 수 있는 가능성이 있습니다.

하지만 calculateTotalResult에 해당하는 코드가 생성자에 담겨져 있다면, GameResult의 result의 상태는 객체 생성 이후에는 변경될수 없기 때문에 일관성을 유지할 수 있습니다.

객체 생성 이후 변하지 않은 값이라면, 생성자 안에서 처리하는것이 좋다고 생각합니다.

Reply:
@kjb95 깨달았습니다..! calculateTotalResult가 결국 setter였군요 그래서 생성자에 넣어야 하고요 감사합니다!

Reply:
저도 많이 배웠습니다..!